src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -149,21 +151,41 @@ class HuiEntitiesCard extends LitElement implements LovelaceCard {
       element.hass = this._hass;
     }
     if (
-      entityConf.entity &&
-      !DOMAINS_HIDE_MORE_INFO.includes(computeDomain(entityConf.entity))
+      (entityConf.tap_action && entityConf.tap_action.action !== "none") ||
+      (entityConf.entity &&
+        !DOMAINS_HIDE_MORE_INFO.includes(computeDomain(entityConf.entity)))
     ) {
       element.classList.add("state-card-dialog");
-      element.addEventListener("click", () => this._handleClick(entityConf));
     }
 
     return html`
-      <div>${element}</div>
+      <div
+        @ha-click=${this._handleClick}
```

Comment:
Same issues like last time. We fight with the embedded element for clicks.

Reply:
But we already we're? In my testing I found no issues with interacting with the embedded elements

Reply:
@balloob is there anything else you want me to verify on this? I don't believe it is an issue.